src/integrations/blocks/breadcrumbs-block.php

```diff
@@ -86,11 +86,14 @@ public function present( $attributes ) {
 		$presenter = new Breadcrumbs_Presenter();
```

Comment:
is DI not available here? could be worth a quick fix.

Reply:
Presenters aren't injectable as they are stateful by design and you may want multiple with different states active within the same request ( for example a simple `Button_Presenter` you may want to create 3 of them with different links and then pass them on to something else that then just gets a list of buttons without having to know any of their details )

Reply:
ah right, it's a presenter. I tripped over the constructor and stopped reading, but this makes sense.